scripts/pre-commit/validate_changelog.py

```diff
@@ -0,0 +1,119 @@
+import subprocess
+import logging
+import re
+import sys
+import typing as tp
+from pathlib import Path
+
+logger = logging.getLogger("validate_changelog_update")
+logger.setLevel(logging.INFO)
+
+handler = logging.StreamHandler()
+handler.setLevel(logging.INFO)
+formatter = logging.Formatter("%(levelname)s - %(message)s")
+handler.setFormatter(formatter)
+logger.addHandler(handler)
```

Comment:
Check the comments in other PR, if we can omit this. If not, ignore this comment.